homeassistant/helpers/device_registry.py

```diff
@@ -828,13 +828,14 @@ def async_get_or_create(
             via_device_id = UNDEFINED
 
         if isinstance(entry_type, str) and not isinstance(entry_type, DeviceEntryType):
-            report(  # type: ignore[unreachable]
+            report_usage(  # type: ignore[unreachable]
```

Comment:
Do we even need to keep this code? 🤔

Reply:
I think it might be time to just clean it up at this point.